README.md

```diff
@@ -40,62 +39,79 @@ When the reader has completed this Code Pattern, they will understand how to:
 
 # Steps
 
-Use the ``Deploy to IBM Cloud`` button **OR** create the services and run locally.
+We'll have two methods to run this application: [using an easy one-click deploy option](#using-the-deploy-to-ibm-cloud-button), or [locally with a few CLI tools](#run-the-application-locally). Depending on your objective, learning or seeing the app quickly, choose accordingly.
 
-## Deploy to IBM Cloud
+## Using the `Deploy to IBM Cloud` button
 
-[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/watson-voice-bot.git)
+> If you prefer to deploy the application manually, scroll down to the next section.
+
+Steps:
+
+1. [Clone the repo](#1-clone-the-repo)
+2. [Create the services and deploy the web app](#2-create-the-services-and-deploy-the-web-app)
+3. [Upload the Watson Assistant workspace](#3-upload-the-watson-assistant-workspace)
+4. [Configure environment variables](#4-configure-environment-variables)
+
+### 1. Clone the repo
+
+Clone the `watson-voice-bot` repo locally. In a terminal, run:
+
+```
+git clone https://github.com/IBM/watson-voice-bot
+```
 
-1. Press the above ``Deploy to IBM Cloud`` button and then click on ``Deploy``.
+We’ll be using the file [`data/workspace.json`](data/workspace.json).
 
-2. In Toolchains, click on `Delivery Pipeline` to watch while the app is deployed. Once deployed, the app can be viewed by clicking `View app`.
+### 2. Create the services and deploy the web app
 
-   ![](doc/source/images/toolchain-pipeline.png)
+The next step is to deploy the application, we'll do this by running clicking the button below. A nice byproduct of doing doing this is that the services required (Speech-to-Text, Text-to-Speech, and Assistant) will be created automatically. Click the button below.
 
-3. To see the app and services created and configured for this Code Pattern, use the IBM Cloud dashboard. The app is named `CodePattern_VoiceBot-` with a unique suffix. The following services are created and easily identified by the `wvb-` prefix:
+[![Deploy to IBM Cloud](https://bluemix.net/deploy/button.png)](https://bluemix.net/deploy?repository=https://github.com/IBM/watson-voice-bot.git)
+
+When the DevOps Toolchain Pipeline appears, click on the `Deploy` button to start the deployment. You can optionally, click on the `Delivery Pipeline` to watch the logs as the app is deployed.
+
+![](doc/source/images/toolchain-pipeline.png)
 
-   * wvb-speechtotext
-   * wvb-texttospeech
-   * wvb-assistant
+Once deployed, the app can be viewed by clicking `View app`. The app can also be viewed in the dashboard. The app should be prefixed with the string `watson-voice-bot-`, and the corresponding services that were created and easily identified by the `wvb-` prefix, i.e.: `wvb-assistant`.
```

Comment:
and the corresponding services that were created can easily be ...

Reply:
done